css/app.css

```diff
@@ -35,3 +35,28 @@ gaia-list .destructive {
 a.navigation *, a.action *, a.edit * {
   pointer-events: none;
 }
+
+gaia-list span.group {
```

Comment:
instead of "group", I'd use something more generic like "title" or "item-title". What do you think? So that it can be reused